packages/api/portfolios/application/getApplications.test.ts

```diff
@@ -0,0 +1,38 @@
+import { handler } from "../application/getApplications";
+import { Context } from "aws-lambda";
+import { ApiGatewayEventParsed } from "../../utils/eventHandlingTool";
+import { ApiSuccessResponse, SuccessStatusCode } from "../../utils/response";
+import { ErrorStatusCode } from "../../../email/utils/statusCodesAndErrors";
```

Comment:
Looks like we got some duplicated enums in the email package.
```suggestion
import { ApiSuccessResponse, SuccessStatusCode, ErrorStatusCode } from "../../utils/response";
```